neo/Network/RPC/RpcServer.cs

```diff
@@ -297,7 +297,21 @@ private JObject Process(string method, JArray _params)
                         return json;
                     }
                 case "getrawmempool":
-                    return new JArray(Blockchain.Singleton.GetMemoryPool().Select(p => (JObject)p.Hash.ToString()));
+                    {
```

Comment:
This change affect API implementations, maybe we should change the return logic with a `format` parameter or something like that

Reply:
The format is the exact same if you don't pass the new parameter. I don't think any change is needed from how it is now.

Reply:
See my comment on line 303; how does that not return the same thing after this change as was returned before this change?